src/tests/unit/data/cic-events.ts

```diff
@@ -147,3 +147,53 @@ export const RESOURCE_NOT_FOUND = {
 	resource: "/invalid",
 	stageVariables: {},
 };
+
+export const VALID_SESSION = {
```

Comment:
pls move to a separate session specific file

Reply:
Done